org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -278,7 +278,7 @@ SsoDeveloper:
       - 'arn:aws:iam::aws:policy/PowerUserAccess'
       - 'arn:aws:iam::aws:policy/AdministratorAccess-AWSElasticBeanstalk'
       - 'arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess'
-      - !CopyValue [ !Sub '${primaryRegion}-cost-explorer-access-policy-ManagedPolicyArn' ]
+    customerManagedPolicyName: !Ref policyCostExplorerSid
```

Comment:
i think the reference needs to be the policy name?  I don't think a policy's Sid is the same as a policy's name.  You probably need to output the policy name in the template that created the policy then reference it here.

Reply:
Oh, hrm, I was interpreting that as the name that AWS uses, not the name that CloudFormation uses. I wonder which it is

Reply:
I believe the Sid is just a description of the policy

Reply:
After some testing I found that we need to set the ManagedPolicyName parameter in the template and use that value. I've updated the dependent PR to also add support for that parameter as well.